camayoc/tests/qpc/cli/test_credentials.py

```diff
@@ -656,8 +664,9 @@ def test_clear_all(isolated_filesystem, qpc_server_config):
         encoding="utf-8",
         withexitstatus=True,
     )
+    successfully_deleted_count = number_of_credentials + new_credentials_count
     assert (
-        f"Successfully deleted {expected_credential_count} credentials. "
+        f"Successfully deleted {successfully_deleted_count} credentials. "
```

Comment:
Would it make sense to use regular expression here? To extract a number, cast it to int and assert it is at least as large as `expected_credential_count`? This way you wouldn't have to list creds earlier and that should be robust against credentials that can't be removed due to them being dependencies.

But I guess we would need to do something about the next line, still...